clients/python/llmengine/data_types.py

```diff
@@ -131,17 +131,27 @@ class CreateLLMEndpointRequest(BaseModel):
     # LLM specific fields
     model_name: str
     source: LLMSource = LLMSource.HUGGING_FACE
-    inference_framework: LLMInferenceFramework = LLMInferenceFramework.DEEPSPEED
+    inference_framework: LLMInferenceFramework = LLMInferenceFramework.TEXT_GENERATION_INFERENCE
```

Comment:
Is this a breaking change? Seems like we're changing default functionality. Might be fine.

Reply:
i don't think people could run build deepspeed models right now